audapter_viewer/calc_newAudapterData.m

```diff
@@ -52,17 +52,18 @@
 %% Set up OST files
 
 % Get the name of the OST, take out any .ost extensions if they exist
-if strcmp(audFileLoc, 'experiment_helpers') || strcmp(audFileName, 'measureFormants')
-    trackingPath = fullfile(get_gitPath('free-speech'), 'experiment_helpers'); 
-elseif isfolder(audFileLoc)
-    if contains(audFileLoc,'/') || contains(audFileLoc,'\') %ie is a full filepath already
-        trackingPath = audFileLoc;
-    else
-        trackingPath = fullfile(get_gitPath('current-studies'), audFileLoc);
-    end
-else
-    trackingPath = fullfile(get_gitPath('current-studies'), audFileLoc); 
-end
+trackingPath = get_trackingFilePath(audFileLoc, audFileName); 
+% if strcmp(audFileLoc, 'experiment_helpers') || strcmp(audFileName, 'measureFormants')
+%     trackingPath = fullfile(get_gitPath('free-speech'), 'experiment_helpers'); 
+% elseif isfolder(audFileLoc)
```

Comment:
@cwnaber I don't know if there is some logic of using isfolder instead of just looking for fileseps? I took it out of the modular one because I didn't see what it was adding